### PR TITLE
Worked on adding dynamic routing to attractions list page + other tasks

### PR DESCRIPTION
**Pull request by rushanniazi**
I worked on adding dynamic routing functionality to the attraction details page, there is just 1 small step remaining to make it ideal. Also, I added logic to the recommend controller to consider less than AND equal to the max guests and opening data/closing date since it was only less than before and we ran into the issue where some attractions were not showing up correctly. Also made a few more fields dynamic on the attractions list page

### CODE REVIEW

File: travvy/src/components/AttractionsList.vue [hunk 1]
```diff
@@ -6,16 +6,16 @@
       <!-- include the toronto heading, an image, and some info about Toronto -->
       <br><br><br>
       <!-- Generates location heading based on first recommended attraction -->
-      <h1>{{$store.state.city}}</h1>
+      <h1>{{$store.state.city}}, {{$store.state.recommendedAttractions[0].country}}</h1>
       <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
       <br>
       <h3>Description</h3>
-      <p>Toronto is the largest city in Canada, is the capital of Ontario, and is home to attractions such as the CN tower.</p>
+      <p>{{$store.state.recommendedAttractions[0].city}} is the largest city in {{$store.state.recommendedAttractions[0].country}}, and is home to attractions such as the {{$store.state.recommendedAttractions[0].attraction_name}}, {{$store.state.recommendedAttractions[1].attraction_name}}, and {{$store.state.recommendedAttractions[2].attraction_name}}.</p>
```
Comment (lkdfount) [inline travvy/src/components/AttractionsList.vue:13]: We may still want to change this at some point. Not all of our cities are the largest for their countries

**Review by lkdfount — APPROVED** [review]
We still have a few more things on the attraction details to make dynamic but this looks awesome Rushan thanks for starting it :)

File: travvy/src/components/AttractionsList.vue (hunk 1 above)
Comment (lucasungaretti) [reply travvy/src/components/AttractionsList.vue:13]: agree, maybe we can have a variable for the entire sentence? aka description of each city

File: travvy-server/src/controllers/RecommendController.js [hunk 2]
```diff
@@ -12,9 +12,10 @@ module.exports = {
         model: Attractions,
         as: 'Attractions',
         where: {city: city,
-                max_number_guest: {[Op.gt]: groupSize}, 
-                opening_date:{[Op.lt]:startDate}, 
-                closing_date:{[Op.gt]: endDate} 
+
+                max_number_guest: {[Op.gte]: groupSize},
```
Comment (dyampols) [inline travvy-server/src/controllers/RecommendController.js:16]: this looks good, thank you for noticing

**Review by dyampols — APPROVED** [review]
Looks great!

File: travvy/src/components/AttractionDetails.vue [hunk 3]
```diff
@@ -3,31 +3,47 @@
...
-    <h3>Maximum Group Size: {{$store.state.recommendedAttractions[0].max_number_guest}}</h3>
-    <h3>Are masks required: {{$store.state.recommendedAttractions[0].mask_required}}</h3>
-    <h3>COVID-19 Safety Measures: {{$store.state.recommendedAttractions[0].covid_safety_measures}}</h3>
+    <h2>Address: <span>{{$store.state.recommendedAttractions[id].street_address}}</span> </h2>
+    <h3>Email: {{$store.state.recommendedAttractions[id].email_address}}</h3>
+    <h3>Phone Number: {{$store.state.recommendedAttractions[id].phone_number}}</h3>
+    <h3>Ticket Pricing: {{$store.state.recommendedAttractions[id].ticket_price}}</h3>
+    <h3>Maximum Group Size: {{$store.state.recommendedAttractions[id].max_number_guest}}</h3>
+    <h3>Are masks required: {{$store.state.recommendedAttractions[id].mask_required}}</h3>
+    <h3>COVID-19 Safety Measures: {{$store.state.recommendedAttractions[id].covid_safety_measures}}</h3>
```
Comment (emilynairnn) [inline travvy/src/components/AttractionDetails.vue:17]: Thanks for making these dynamic - I think it'll be a great addition to our site!

File: travvy/src/components/AttractionsList.vue [hunk 4]
```diff
@@ -41,9 +43,9 @@
       <!-- in later sprints, they will be pulled from the database -->
       <h1>Traveller Tips</h1>
       <h3>Matt, 25 from Montreal</h3>
-      <p>Toronto felt pretty safe but if you are worried about COVID it may not be the best time to go, it is tough to get around without being around a lot of people.</p>
+      <p>{{$store.state.recommendedAttractions[0].city}} felt pretty safe but if you are worried about COVID it may not be the best time to go, it is tough to get around without being around a lot of people.</p>
       <h3>Lisa, 28 from Winnepeg</h3>
-      <p>I stayed along Harbourfront, I'd totally recommend it. Great view and less people.</p>
+      <p>I spent time at {{$store.state.recommendedAttractions[0].attraction_name}}, I'd totally recommend it. Great view and less people.</p>
```
Comment (emilynairnn) [inline travvy/src/components/AttractionsList.vue:48]: Another key thing to make dynamic!

**Pull request merged**